px-datetime-behavior.html

```diff
@@ -1,114 +1,108 @@
-<link rel="import" href="px-datetime-shared-behavior.html" />
+<link rel="import" href="px-datetime-shared-behavior.html"/>
 
 <script>
+(function() {
+  var PxDatetimeBehavior = window.PxDatetimeBehavior = (window.PxDatetimeBehavior || {});
 
-/**
-  ```pxDatetimeBehavior```
-
-  This behavior is for all ```px-{datetime}-{picker}``` components.
-
-  Dependencies: momentjs
+  /**
+   * For all `px-{datetime}-{picker}` components.
+   * Dependencies: momentjs
+   *
+   * @polymerBehavior PxDatetimeBehavior.SingleDate
+   */
+  PxDatetimeBehavior.SingleDate = [{
```

Comment:
Should this be `.SingleMoment` to match the other?

Reply:
good idea